### ISSUE TITLE
Setting output directory only directs the .cue file when using --split?

### ISSUE THREAD

**Issue by Silvia-Rose** [putnam/binmerge#21]
What I was aiming for was to take the .cue and .bin in "input" folder, do the splitting, and then output **both** the .cue and .bin(s) to the "output" folder.

My batch is as follows:

```
set "CueFileName=%%~ni"
for %%i in (input\*.cue) do (binmerge -o output -s "%%i" "%CueFileName%")
```

and this does correctly split the files, however it only puts the new .cue file in the "output" folder, and just spits the .bin files next to the binmerge.exe rather than also in "output".

I'm not well versed so I'm not sure if it's because of my batch or if this is unintended behaviour?

I appreciate your program 💜

**Comment by putnam** [comment]
Oh, yeah, that's definitely broken in split mode. Will take a look.

**Comment by putnam** [comment]
Yeah, this is me not testing properly with the 1.0.2 release. If you roll back to 1.0.1 it will work as expected. I'll fix this and cut a new release.

**Comment by putnam** [comment]
Fixed in release 1.0.3
